citesphere/src/main/java/edu/asu/diging/citesphere/core/authority/impl/ViafAuthorityImporter.java

```diff
@@ -86,9 +87,21 @@ public IImportedAuthority retrieveAuthorityData(String uri) throws URISyntaxExce
         }
         return null;
     }
-    
+
     @Override
     public String getId() {
         return ID;
     }
+
+    @Override
+    public AuthoritySearchResult searchAuthorities(String firstName, String lastName, int page, int pageSize)
+            throws AuthorityServiceConnectionException {
+
+        return null;
```

Comment:
same as above, let's throw a not yet implemented exception, sine this will be added soon.